app/src/main/java/com/wednesday/banklocator/model/IfscResponse.kt

```diff
@@ -1,15 +1,55 @@
-package com.wednesday.banklocator.model
-
-data class IfscResponse(
-    val ADDRESS: String,
-    val BANK: String,
-    val BRANCH: String,
-    val CITY: String,
-    val CONTACT: String,
-    val DISTRICT: String,
-    var IFSC: String,
-    val RTGS: Boolean,
-    val STATE: String
-)
-
-
+package com.wednesday.banklocator.model
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class IfscResponse(
+    var ADDRESS: String="",
+    var BANK: String="Search",
+    var BRANCH: String="",
+    var CITY: String="",
+    var CONTACT: String="",
+    var DISTRICT: String="",
+    var IFSC: String="",
+    var RTGS: Boolean=false,
+    var STATE: String=""
+):Parcelable {

```

Comment:
Why is this needed?

Reply:
Actually, we need Parcelable to pass information through the 'model class'.